glide.lock

```diff
@@ -58,7 +95,7 @@ imports:
   - version
   - wire
 - name: go.uber.org/yarpc
-  version: 6ae533f0810337028ef055b690360e050aa13219
+  version: 6b9245d126758870690c44f27f695cc7e811f913
```

Comment:
I think I'm going to pull the glide update out into a separate commit and address this issue as well.